### PR TITLE
Add new Chocobar for Love

### PR DESCRIPTION
**Pull request by rohitjakhar**
Added a new chocobar for showing love

### CODE REVIEW

**Comment by Pradyuman7** [comment]
@rohitjakhar great work, but could you also add screenshot of the new Snackbar in the ReadMe, it will help the users to see the new additions.

File: app/src/main/java/com/pd/snickers/MainActivity.java [hunk 1]
```diff
@@ -81,6 +81,13 @@ public void onClick(View v) {
             }
         });
 
+        findViewById(R.id.button_love).setOnClickListener(new View.OnClickListener() {
```
Comment (Pradyuman7) [inline app/src/main/java/com/pd/snickers/MainActivity.java:84]: Could you also add a screenshot of this Snacker (our Chocobar) in the ReadMe

**Review by Pradyuman7 — CHANGES_REQUESTED** [review]

**Comment by rohitjakhar** [comment]
Update README.md file

File: app/src/main/java/com/pd/snickers/MainActivity.java (hunk 1 above)
Comment (rohitjakhar) [reply app/src/main/java/com/pd/snickers/MainActivity.java:84]: > Could you also add a screenshot of this Snacker (our Chocobar) in the ReadMe

Yes, Added Done

**Review by Pradyuman7 — APPROVED** [review]

**Comment by Pradyuman7** [comment]
Great work @rohitjakhar 🎉

**Comment by rohitjakhar** [comment]
> Great work @rohitjakhar tada
Thanks Bro @Pradyuman7  this is my first contribution in android library

**Pull request merged**